Runtime/Android/QonversionWrapperAndroid.cs

```diff
@@ -150,8 +150,21 @@ public void SetNotificationsToken(string token)
             CallQonversion("setNotificationsToken", token);
         }
 
+        public bool HandleNotification(string notification)
+        {
+            return CallQonversionBool("handleNotification", notification);
+        }
+
         private const string QonversionWrapper = "com.qonversion.unitywrapper.QonversionWrapper";
 
+        private static bool CallQonversionBool(string methodName, params object[] args)
```

Comment:
Is it possible to use jeneric type here so we can use this method with other type in future?